sources/packages/api/src-sql/Restrictions/Insert-restrictions.sql

```diff
@@ -0,0 +1,1292 @@
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_type [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'SA',
+    'Stop All Assessment-See user for details.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_type [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'SO',
+    'SOG Restriction - stop docs producing.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_type [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'SP',
+    'Students with Permanent Disability.',
+    'Federal',
+    ARRAY ['No effect'] :: sims.restriction_action_type [],
+    'No effect'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'IC',
+    'ICP - Application on file.',
+    'Federal',
+    ARRAY ['No effect'] :: sims.restriction_action_type [],
+    'Warning'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'IS',
+    'ICP file - to Appeals for review.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_type [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'F3',
+    'F1-5 restriction period varies from 1 - 5 years based on severity of the fraud.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_type [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'F4',
+    'F1-5 restriction period varies from 1 - 5 years based on severity of the fraud.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_type [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'F5',
+    'F1-5 restriction period varies from 1 - 5 years based on severity of the fraud.',
```

Comment:
The value here is the same as the ones in line 1155 and 1136. For my understanding, are they all supposed to be the same ?

Reply:
I think code F1 to F5 is the same description, I copied the details from the doc file, you can check the Sharepoint `General->Analysis->Restrictions`